modelling/src/main/java/org/axonframework/modelling/entity/PolymorphicEntityModelBuilder.java

```diff
@@ -36,8 +36,8 @@ public interface PolymorphicEntityModelBuilder<E> extends EntityModelBuilder<E>
 
     @Nonnull
     @Override
-    PolymorphicEntityModelBuilder<E> commandHandler(@Nonnull QualifiedName qualifiedName,
-                                                    @Nonnull EntityCommandHandler<E> messageHandler);
+    PolymorphicEntityModelBuilder<E> instanceCommandHandler(@Nonnull QualifiedName qualifiedName,
```

Comment:
Shouldn't this builder also override the `creationalCommandHandler` method, to ensure a `PolymorphicEntityModelBuilder` is returned?

Reply:
You are right! Adding